test/TensorFlow/diagnostics.swift

```diff
@@ -49,7 +49,8 @@ public func testDevice() {
 // should be a single copy-to-host compiler warning.
 public func SR8412_CopyToHost() {
   for _ in 0...10 {
-    let x = Tensor(1)  // expected-warning {{value implicitly copied to the host}}
+		// This gets moved outside the loop by the compiler optimizations. So, no warnings. 
```

Comment:
tabs are used here and in some other places in this patch

Reply:
Oops...fixed now.